src/jit/gentree.cpp

```diff
@@ -16885,21 +16884,28 @@ GenTreeSIMD* Compiler::gtNewSIMDNode(
 GenTreeSIMD* Compiler::gtNewSIMDNode(
     var_types type, GenTreePtr op1, GenTreePtr op2, SIMDIntrinsicID simdIntrinsicID, var_types baseType, unsigned size)
 {
-    // TODO-CQ: An operand may be a GT_OBJ(GT_ADDR(GT_LCL_VAR))), in which case it should be
-    // marked lvUsedInSIMDIntrinsic.
     assert(op1 != nullptr);
     if (op1->OperIsLocal())
     {
```

Comment:
On line 16871 we are checking whether op1->OperGet() == GT_LCL_VAR whereas here using OperIsLocal().  I think here too we should use op1->OperIsLocal()?